tritium-caffeine/src/main/metrics/cache-metrics.yml

```diff
@@ -0,0 +1,38 @@
+options:
+  javaPackage: com.palantir.tritium.metrics.caffeine
+  javaVisibility: packagePrivate
+namespaces:
+  cache:
+    docs: Cache statistic metrics
+    metrics:
+      hit.count:
+        type: counter
+        tags: [cache]
+        docs: Count of cache hits
```

Comment:
Sorry I missed this in the previous round of review, I think all the counters should be updated to `type: meter`. Counter is interpreted as a gauge in most backends because it doesn't guarantee monotonic increasing values, where our Meter, confusingly, uses the [Prometheus Counter type](https://prometheus.io/docs/concepts/metric_types/#counter) for slightly easier indexing.

We may want to remove the `.count` name suffixes as well (otherwise we'll end up with queries for `cache_hit_count_count`), since those are described by both the `counter` and `meter` metric types.

Reply:
yeah, I'm trying to keep metric name compatibility with the existing metrics produced by `com.palantir.tritium.metrics.caffeine.CaffeineCacheStats#registerCache(com.palantir.tritium.metrics.registry.TaggedMetricRegistry, com.github.benmanes.caffeine.cache.Cache<?,?>, java.lang.String)` for now as I'm not sure how complex the migration will be across the fleet.